webpack.config.js

```diff
@@ -0,0 +1,32 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.js',
+  devServer: {
+    static: './dist',
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+
+  optimization: {
+    runtimeChunk: 'single',
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+};
```

Comment:
- [OPTIONAL] when I run `npm start`, everything working fine, but there is a warning appear says that you should Set 'mode' option to 'development' or 'production' to enable defaults for each environment.

> So to fix this warning 💡 Just add ` mode: 'development',` to your `module.exports ` object 👍

![image](https://user-images.githubusercontent.com/45231276/195259446-f114fd5c-395d-49e2-95b5-951472352a65.png)

Reply:
Hello, @Bondok6 
Thanks very much for the review.
I have been trying to fix that.
Thanks for showing me how to fix it